README.adoc

```diff
@@ -376,9 +389,8 @@ Constraint Violation Found: must be greater than or equal to 18
 Constraint Violation Found: must be a well-formed email address
 ----
 
-To try the method-level validation expand the `/beanvalidation/launchspacecraft
-POST request to specify a launch code` section.  Enter `OpenLiberty` in the text box
-to note that `launched` is returned because the launch code passes the defined
+To try the method-level validation expand the `/beanvalidation/launchspacecraft POST request to specify a launch code` section.
+Enter `OpenLiberty` in the text box to note that `launched` is returned because the launch code passes the defined
```

Comment:
Enter `OpenLiberty` in the text box to note that `launched` is returned because the launch code passes the defined...
--->
Enter `OpenLiberty` in the text box. Note that `launched` is returned because the launch code passes the defined...